.gitignore

```diff
@@ -1,5 +1,6 @@
 .idea
 node_modules/
 dist/app
+.env
 *.log
 .application-config.json
```

Comment:
is this still necessary to ignore?

Reply:
Yes, the documentation strongly recommend not to commit .env file.

Reply:
good answer, but it was not a tricky question 😄  i was thinking about the `.application-config.json`